Frameworks/include/CoreGraphics/D2DWrapper.h

```diff
@@ -36,6 +36,10 @@ inline D2D_POINT_2F _CGPointToD2D_F(CGPoint point) {
     return { point.x, point.y };
 }
 
+inline CGPoint _D2DPointToCGPoint(D2D_POINT_2F point) {
+    return CGPointMake(point.x, point.y);
```

Comment:
but directly above it CG->D2D uses the other convention.

---
In reply to: [90985613](https://github.com/Microsoft/WinObjC/pull/1507#discussion_r90985613) [](ancestors = 90985613,90974360)